apps/condo/domains/organization/integrations/sbbol/sync/requestTransactions.js

```diff
@@ -17,7 +17,7 @@ const { ERROR_PASSED_DATE_IN_THE_FUTURE } = require('../constants')
 
 
 const logger = getLogger('sbbol/SbbolSyncTransactions')
-const isTransactionsReceived = (response) => (get(response, 'error.cause', '') !== 'STATEMENT_RESPONSE_PROCESSING')
+const isResponceSuccessfull = (response) => (get(response, 'error.cause', '') !== 'STATEMENT_RESPONSE_PROCESSING')
```

Comment:
Since you are checking on processing status of the response, then maybe to use according name for the function?
```suggestion
const isResponceProcessing = (response) => (get(response, 'error.cause', '') !== 'STATEMENT_RESPONSE_PROCESSING')
```